internal/flusher/flusher.go

```diff
@@ -0,0 +1,43 @@
+package flusher
+
+import (
+	"github.com/ozonva/ova-location-api/internal/location"
+	"github.com/ozonva/ova-location-api/internal/repo"
+	"github.com/ozonva/ova-location-api/internal/utils"
+)
+
+type Flusher interface {
+	Flush(entities []location.Location) []location.Location
+}
+
+func NewFlusher(
+	chunkSize int,
+	entityRepo repo.Repo,
+) Flusher {
+	return &flusher{
+		chunkSize:  chunkSize,
+		entityRepo: entityRepo,
+	}
+}
+
+type flusher struct {
+	chunkSize  int
+	entityRepo repo.Repo
+}
+
+func (flusher *flusher) Flush(entities []location.Location) []location.Location {
+	chunks := utils.LocationSliceSplit(entities, flusher.chunkSize)
+	unsavedEntities := make([]location.Location, 0, len(entities))
```

Comment:
Это та редкая ситуация, когда возможно лучше сделать так:
```suggestion
	var unsavedEntities []location.Location
```
Потому что ты капэсити всё равно не знаешь заранее – и не узнаешь никак